grammar.js

```diff
@@ -572,7 +588,7 @@ module.exports = grammar({
 
     struct_declaration: $ => seq(
       repeat($.attribute_list),
-      repeat($.modifier),
+      repeat(alias($.modifier_with_ref, $.modifier)),
```

Comment:
I think we can just go with `repeat($.modifier)` here followed by an `optional('ref')` before the `struct` as it looks like it has to be the last modifier before struct anyway.

Reply:
Thanks. Sounds good. I've modified this and the above in https://github.com/tree-sitter/tree-sitter-c-sharp/pull/251/commits/08fecfd9b2ccc9743f70968136804626b5e05bbc.